recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "snowflake-snowpark-python" %}
-{% set version = "1.21.0" %}
+{% set version = "1.22.0" %}
```

Comment:
seems like the cut was made from an older version. This should be `1.21.1`?

Reply:
I have https://github.com/snowflakedb/snowpark-python/pull/2232 pending now (will be merged so once CI is green) - let's have 1.22.0 release based on this change

Reply:
sorry, @sfc-gh-jdu what do you mean, do you mean leave it like 1.21.0 -> 1.22.0?